reference/configuration/security.rst

```diff
@@ -265,7 +265,7 @@ you may create a redirect loop. For details, see
 check_path
 ..........
 
-**type**: ``string`` **default**: ``/login_check``
+**type**: ``string`` **default**: ``login``
```

Comment:
this (and above) is wrong, the default set in the MainConfiguration class of SecurityBundle is still `/login_check` and `/login`